### PR TITLE
refactored Account component

### PR DESCRIPTION
**Pull request by a-merezhanyi**
- renamed folder to be concise with the component and naming
- resolved lint issues
- redirected `healthcheck` to `/`

### CODE REVIEW

File: src/main/java/nextmainfocus/account/Account.java [hunk 1]
```diff
@@ -39,4 +39,9 @@ public boolean isCredentialsNonExpired() {
 	public boolean isEnabled() {
 		return true;
 	}
+
+	@Override
+	public String toString() {
+		return "Account [id=" + id + ", username=" + username + ", password=" + password + ", role=" + role + "]";
```
Comment (merezhanyi) [inline src/main/java/nextmainfocus/account/Account.java]: why do we need this? and how the password will look like (encypet or not)?

File: src/main/java/nextmainfocus/account/AccountController.java [hunk 2]
```diff
@@ -22,76 +19,39 @@
...
 @RequestMapping("api/v1/")
-public class SecurityController {
+public class AccountController {
 
-	private static final Logger logger = LoggerFactory.getLogger(SecurityService.class);
+	private static final Logger logger = LoggerFactory.getLogger(AccountController.class);
 
 	@Autowired
-	SecurityService securityService;
+	AccountService securityService;
```
Comment (merezhanyi) [inline src/main/java/nextmainfocus/account/AccountController.java]: are these best practices (the renaming)? such renaming may be confusing..

File: src/main/java/nextmainfocus/account/AccountController.java [hunk 3]
```diff
@@ -22,76 +19,39 @@
...
-			JSONObject body = new JSONObject();
-			body.put("code", "INTERNAL_SERVER_ERROR");
-			body.put("message", "Oops! User creation failed.");
-			status = HttpStatus.INTERNAL_SERVER_ERROR;
+	private UserDetails getUserDetailsWithEncodedPassword(Account account) {
+		return org.springframework.security.core.userdetails.User.withUsername(account.getUsername())
+				.password(passwordEncoder.encode(account.getPassword())).roles(account.getRole()).build();
+	}
 
-			logger.error(account.getUsername() + " was not created due to:");
```
Comment (merezhanyi) [inline src/main/java/nextmainfocus/account/AccountController.java:85]: do we still have logging of errors somewhere?

File: src/main/java/nextmainfocus/account/SecurityConfiguration.java [hunk 4]
```diff
@@ -44,10 +44,10 @@ public InMemoryUserDetailsManager userDetailsManager() {
 	}
 
 	@Bean
-	public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
-		http.authorizeHttpRequests().requestMatchers("/healthcheck", "/api/v1/login/").permitAll().anyRequest()
+	SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
+		http.authorizeHttpRequests().requestMatchers("/",
+				"/api/v1/login/").permitAll().anyRequest()
```
Comment (merezhanyi) [inline src/main/java/nextmainfocus/account/SecurityConfiguration.java:49]: should the url be hard-coded at this level?

File: src/main/java/nextmainfocus/healthcheck/HealthCheckController.java [hunk 5]
```diff
@@ -12,10 +12,10 @@ public class HealthCheckController {
 	private static final Logger logger = LoggerFactory.getLogger(HealthCheckController.class);
 
 	@CrossOrigin(origins = "*")
-	@GetMapping("/healthcheck")
+	@GetMapping("/")
```
Comment (merezhanyi) [inline src/main/java/nextmainfocus/healthcheck/HealthCheckController.java:15]: why healthcheck was moved to `/` ? 
best practices?

File: src/main/java/nextmainfocus/account/AccountController.java (hunk 3 above)
Comment (a-merezhanyi) [reply src/main/java/nextmainfocus/account/AccountController.java:85]: This message is present. Methods are rearranged.

File: src/main/java/nextmainfocus/account/SecurityConfiguration.java (hunk 4 above)
Comment (a-merezhanyi) [reply src/main/java/nextmainfocus/account/SecurityConfiguration.java:49]: Yes, afaik.

File: src/main/java/nextmainfocus/healthcheck/HealthCheckController.java (hunk 5 above)
Comment (a-merezhanyi) [reply src/main/java/nextmainfocus/healthcheck/HealthCheckController.java:15]: To have something at the root level. It might be replaced some day.

**Pull request merged**